Panic when the cache is empty

```
$ gh not -c config.local.yaml 
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/nobe4/gh-not/internal/notifications.Notifications.Render({0x105074600, 0x0, 0x104a45ebe?})
	/home/runner/work/gh-not/gh-not/internal/notifications/view.go:120 +0x780
github.com/nobe4/gh-not/internal/cmd.display({0x105074600, 0x0, 0x0})
	/home/runner/work/gh-not/gh-not/internal/cmd/root.go:156 +0x38
github.com/nobe4/gh-not/internal/cmd.runRoot(0x140001da500?, {0x104a3e957?, 0x7?, 0x104a3d2ed?})
	/home/runner/work/gh-not/gh-not/internal/cmd/root.go:97 +0x164
github.com/spf13/cobra.(*Command).execute(0x105005ee0, {0x1400001e160, 0x2, 0x2})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:985 +0x840
github.com/spf13/cobra.(*Command).ExecuteC(0x105005ee0)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1117 +0x344
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1041
github.com/nobe4/gh-not/internal/cmd.Execute(...)
	/home/runner/work/gh-not/gh-not/internal/cmd/root.go:48
main.main()
	/home/runner/work/gh-not/gh-not/cmd/gh-not/main.go:13 +0x24
```

--- a/internal/cache/cache.go
+++ b/internal/cache/cache.go
@@ -66,7 +66,7 @@ func (c *FileCache) RefreshedAt() time.Time {
 		// Returning a valid time.Time that's the 0 epoch allows to not return
 		// an error but still process the Expiration date logic correctly.
 		slog.Warn("Could not read file info", "file", c.path, "error", err)
-		return time.Date(0, 0, 0, 0, 0, 0, 0, nil)
+		return time.Unix(0, 0)
 	}
 
 	return info.ModTime()

--- a/internal/notifications/view.go
+++ b/internal/notifications/view.go
@@ -85,6 +85,10 @@ func (n Notification) Visible() bool {
 // Render the notifications in a human readable format.
 // If possible, render a table, otherwise render a simple string.
 func (n Notifications) Render() error {
+	if len(n) == 0 {
+		return nil
+	}
+
 	// Default to a simple string
 	for _, n := range n {
 		n.rendered = fmt.Sprintf("%s %s %s %s by %s at %s: '%s'", n.prettyRead(), n.prettyType(), n.prettyState(), n.Repository.FullName, n.Author.Login, text.RelativeTimeAgo(time.Now(), n.UpdatedAt), n.Subject.Title)

